src/notifier/discord_notifier.py

```diff
@@ -0,0 +1,44 @@
+# std
+import http.client
+import logging
+import urllib.parse
+import socket
+from typing import List
+
+# project
+from . import Notifier, Event, EventType
+
+
+class DiscordNotifier(Notifier):
+    def __init__(self, title_prefix: str, config: dict):
+        logging.info("Initializing Discord notifier.")
+        super().__init__(title_prefix, config)
+        try:
+            self.webhook_url = config["webhook_url"]
+        except KeyError as key:
+            logging.error(f"Invalid config.yaml. Missing key: {key}")
+
+    def send_events_to_user(self, events: List[Event]) -> bool:
+        errors = False
+        for event in events:
+            if event.type == EventType.USER:
+                o = urllib.parse.urlparse(self.webhook_url)
+                conn = http.client.HTTPSConnection(o.netloc)
+                conn.request(
+                    "POST",
+                    o.path,
+                    urllib.parse.urlencode(
+                        {
+                            "username": f"chiadog on {socket.gethostname()}",
```

Comment:
I like the idea of hostname but that is a bit more difficult to control. Let's use the title prefix which is configurable from the config.yaml file. The title prefix approach also works when starting multiple `chiadog` instances on the same machine that monitor a few remote harvesters. See how it's used in the Pushover notifier for example.

Reply:
I also liked @greimela's [suggestion](https://github.com/martomi/chiadog/pull/13#discussion_r612297928) for 🚨 symbol for high priority events. What do you think, would that be valuable on Discord too?

See how we have it now on Telegram [for reference](https://github.com/martomi/chiadog/blob/main/src/notifier/telegram_notifier.py#L30), maybe you can use the same approach.

Reply:
Look cool, I see embeds are also supported through Webhooks API. Perhaps a bit more work, I'm also fine if you just follow the pattern we used for Telegram already 😄

Reply:
Awesome! 👍